### PR TITLE
Initial work for debugging step in chapter 7

### PR DESCRIPTION
**Pull request by v-bpastiu**

### CODE REVIEW

File: 07-alerting-and-notification/README.md [hunk 1]
```diff
@@ -157,4 +159,26 @@ Steps to follow:
...
+
+    ```shell
+    kubectl --namespace monitoring port-forward svc/kube-prom-stack-grafana 3000:80
+    ```
+
+2. Open a web browser on [localhost:3000](http://localhost:3000) and log in using the default credentials (`admin/prom-operator`).
+3. Navigate to the [Alerting](http://localhost:3000/alerting) section
+4. From the `State` filter click on the `Firing` option, identify the `emojivoto-instance-down` alert defined in the [Creating a New Alert](#creating-a-new-alert) section and expand it. You should see the following:
+![Grafana Alert](assets/images/grafana-alert.png)
+5. Click on the `See graph` button. From the next page you can observe the count for the number of pods in the `emojivoto` namespace displayed as a metric.
```
Comment (bikram20) [inline 07-alerting-and-notification/README.md:179]: Is this is metrics graph for the specific resource at that point in time?

File: 07-alerting-and-notification/README.md [hunk 2]
```diff
@@ -157,4 +159,26 @@ Steps to follow:
...
+6. From the `Explore` tab select the `Loki` data source and in the `Log browser` input the following: `{namespace="emojivoto"}` and click on the `Run query` button from the top right side of the page. You should see the following:
```
Comment (bikram20) [inline 07-alerting-and-notification/README.md:180]: If an alerts fired a day or week before, there would be way too many logs by doing a resource search. We should provide the time window option in the query.

Comment (v-bpastiu) [reply 07-alerting-and-notification/README.md:180]: Agreed. I will include that.

File: 07-alerting-and-notification/README.md (hunk 1 above)
Comment (v-bpastiu) [reply 07-alerting-and-notification/README.md:179]: The user will see what the grafana time range is set to. This is defaulted to "Last 1 hour". We can also instruct the user to update that to his current needs. Those could be 1 hour/12 hours/5 minutes. Depends.

**Review by bikram20 — APPROVED** [review]

**Pull request merged**